packages/data-explorer-ui/src/components/common/Banner/banner.styles.ts

```diff
@@ -0,0 +1,16 @@
+import styled from "@emotion/styled";
+import { Banner } from "./banner";
+
+export const BannerPrimary = styled(Banner)`
+  background-color: ${({ theme }) => theme.palette.primary.main};
+  color: ${({ theme }) => theme.palette.common.white};
+  padding: 4px 8px;
+  text-align: center;
+
+  & > a {
```

Comment:
Let's make the Banner children prop completely configurable - now that we have `TextButtonWhite` we no longer need to specify styles for an anchor. I suggest removing lines 10-15.